examples/keypoint_detection/pyproject.toml

```diff
@@ -10,6 +10,8 @@ python = ">=3.8,<3.12"
 kolena = ">=0.69.0,<1"
 s3fs = "^2023.5.0"
 fsspec = "^2023.5.0"
+pydantic = ">=1.8,<2.0"
+
```

Comment:
Super-nit: extra newline

Reply:
Removed the new line but it is not being shown correctly in the diff